pulsar-client-cpp/lib/ClientConnection.cc

```diff
@@ -1557,34 +1567,34 @@ void ClientConnection::close(Result result) {
     }
 
     lock.unlock();
-    LOG_INFO(cnxString_ << "Connection closed");
+    LOG_INFO(cnxString_ << "Connection closed with " << result);
 
     for (ProducersMap::iterator it = producers.begin(); it != producers.end(); ++it) {
-        HandlerBase::handleDisconnection(ResultConnectError, shared_from_this(), it->second);
+        HandlerBase::handleDisconnection(result, shared_from_this(), it->second);
```

Comment:
When this call `handleDisconnection`  and param `result == ResultRetryable`,  `handleDisconnection` method should handle `ResultRetryable`?

https://github.com/apache/pulsar/blob/fb0f653eadcf6bf72eb8c8efcc29975da6e21267/pulsar-client-cpp/lib/HandlerBase.cc#L108-L121

Reply:
Addressed.